packages/react-dom/package.json

```diff
@@ -47,21 +49,27 @@
     ".": "./index.js",
     "./client": "./client.js",
     "./server": {
+      "workerd": "./server.edge.js",
+      "edge-light": "./server.edge.js",
```

Comment:
Isn't `edge-light` supposed to be a "catch-all" for those environments discussed by WinterCG? If yes, why the `workerd` condition is used here if it points to the same file? What about other conditions like `edge-routine`, `netlify`, etc?

Reply:
Is there any condition that acts as a catch-all? I understand that it might not work for your use case here but some other libraries might just want to ship something that works in **any** worker-like environment. The `browser` condition doesn't work well for that right now.

Reply:
Yea, I've looked into that before. Mainly the thing that sucks is that there's no distinction between "serviceworker" and "worker"-like on the server isn't the same a "worker" on the client. E.g. these APIs can't really be used in actual web workers but can be used in "serviceworker". So I've been waiting on further clarification on that before moving further.

I think the notion of these environments just being "a simple worker" is getting to be outdated as more features are added. I suspect it'll be more like a special "wintercg worker" concept or something that's a superset that's useful.

Reply:
It's definitely not always possible to use a single condition to catch them all - but as long as you only need some subset of the functionality, then it makes sense to reuse the condition. 

Do you have any opinions about Next adding the support for `worker` condition as well? It just looks so off to me that it resolves `browser` but not `worker`